src/utils/constants.js

```diff
@@ -169,5 +170,9 @@ export const notification_template = {
   [NOTIFICATION_ID.CONTRIBUTION_RECEIVED]: {
     message: 'You received a contribution to',
     icon: plus
+  },
+  [NOTIFICATION_ID.BOUNTY_COMPLETED]: {
+    message: 'Your bounty ran out of funds',
```

Comment:
Should this be a plus? Can you do a minus?

Reply:
Ha! Yes

Reply:
Done